### PR TITLE
Feat/rating slider

### PR DESCRIPTION
**Pull request by rpietrek**

### CODE REVIEW

**Review by iamnotadamkr — APPROVED** [review]
Well done!

File: cypress/integration/tests_e2e/slider.spec.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+//
+/// <reference types="cypress" />
+
+describe("Slider with assertion", () => {
+  beforeEach(() => {
+    cy.visit("/");
+  });
+
+  it("should visit slider - min and max rating", () => {
```
Comment (bi3ld) [inline cypress/integration/tests_e2e/slider.spec.js:9]: maybe `Slider should be set with default values`

File: cypress/integration/tests_e2e/slider.spec.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    cy.get('[data-test="slider"] .ngx-slider-pointer-min')
+    .should('have.attr', 'aria-valuetext', '0')
+
+    cy.get('[data-test="slider"] .ngx-slider-pointer-max')
+    .should('have.attr', 'aria-valuetext', '5')
+  });
+
+  it("should change value of the elements", () => {
+
+    cy.get('[data-test="slider"] .ngx-slider-pointer-min')
```
Comment (bi3ld) [inline cypress/integration/tests_e2e/slider.spec.js:20]: Those tests checks slider which is actually not our component.
We should test here end to end. So after change range we should check if results are correct.

File: src/app/places/places.component.ts [hunk 3]
```diff
@@ -66,46 +68,49 @@ export class PlacesComponent implements OnInit {
...
       a.Name < b.Name ? 1 : b.Name < a.Name ? -1 : 0
     );
     this.page = 1;
     this.router.navigate(['/'], { queryParams: { order: 'alf-dsc' } });
   }
 
-  private fetchPlaces(): any {
-    this.PlacesFromService = this.PlacesService.getPlaces();
+  rangeChange(event: RangeInterface) {
+    console.log('from places', event);
```
Comment (bi3ld) [inline src/app/places/places.component.ts:109]: Remove `console.log`

File: src/app/places/places.component.ts [hunk 4]
```diff
@@ -66,46 +68,49 @@ export class PlacesComponent implements OnInit {
...
+    this.Places.sort((a, b) =>
       a.Name < b.Name ? 1 : b.Name < a.Name ? -1 : 0
     );
     this.page = 1;
     this.router.navigate(['/'], { queryParams: { order: 'alf-dsc' } });
   }
 
-  private fetchPlaces(): any {
-    this.PlacesFromService = this.PlacesService.getPlaces();
+  rangeChange(event: RangeInterface) {
```
Comment (bi3ld) [inline src/app/places/places.component.ts:108]: I would change this method name to `filterPlaces`

File: src/app/places/places.component.ts (hunk 3 above)
Comment (rpietrek) [reply src/app/places/places.component.ts:109]: Removed :)

File: src/app/places/places.component.ts (hunk 4 above)
Comment (rpietrek) [reply src/app/places/places.component.ts:108]: Changed :-)

**Pull request merged**